Resources/Prototypes/Entities/Virtual/beam.yml

```diff
@@ -0,0 +1,9 @@
+﻿# Special entity used to keep track of beams from the BeamComponent and what targets they collided with
+- type: entity
+  id: VirtualBeamEntityController
+  name: BEAM ENTITY YOU SHOULD NOT SEE THIS
```

Comment:
needs `HideContextMenu` though i dont think it would show up normally either way